nixos/modules/config/users-groups.nix

```diff
@@ -330,6 +330,20 @@ let
           administrator before being able to use the system again.
         '';
       };
+
+      linger = mkOption {
+        type = types.bool;
```

Comment:
I think we should change this to nullOr bool and default to null.

Then we can always deactivate lingering when it is set to false, activate it when it is true and leave it like it is when it is null.